src/get-bounding-client-rect.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isConnectedNode} from './dom';
+import {layoutRectLtwh} from './layout-rect';
+
+const nativeClientRect = Element.prototype.getBoundingClientRect;
+
+/**
+ * Polyfill for Node.getBoundingClientRect API.
+ * @param {!Element} el
+ * @return {!LayoutRectDef}
+ */
+function getBoundingClientRect(el) {
+  let result = layoutRectLtwh(0, 0, 0, 0);
+
+  if (isConnectedNode(el)) {
+    result = nativeClientRect.call(el);
+  }
+
+  return result;
+}
+
+/**
+ * Determines if this polyfill should be installed.
+ * @param {!Window} win
+ * @return {boolean}
+ */
+function shouldInstall(win) {
+  try {
+    const div = win.document.createElement('div');
+    div.getBoundingClientRect();
```

Comment:
This will likely be DCE'd by closure compiler. Instead, we could try something like:

```js
const rect = div.getBoundingClientRect();
return div.top !== 0;
```

Reply:
@delima02 What does the compiled output of this look like? 

Hint: use `gulp dist --noextensions` and then try to find this code snippet inside `dist/v0.js`. 

You can open the file in Chrome and use devtools pretty printing to make life easier: https://developers.google.com/web/tools/chrome-devtools/javascript/pretty-print